src/pspm_convert_ppg2hb.m

```diff
@@ -61,9 +61,13 @@
 % -------------------------------------------------------------------------
 if nargin < 1
   warning('ID:invalid_input', 'No input. Don''t know what to do.'); return;
-elseif nargin < 2 
-    options = struct();
+elseif ~ischar(fn)
+  warning('ID:invalid_input', 'Need file name string as first input.'); return;
```

Comment:
The check `~ischar(fn)` should be removed here. this is checked in `pspm_load_data` in a coherent way.

Reply:
Yes.. I have removed the condition of `~ischar(fn)` here now.